zcash_client_memory/src/wallet_write.rs

```diff
@@ -935,6 +935,10 @@ impl<P: consensus::Parameters> WalletWrite for MemoryWalletDb<P> {
         Ok(())
     }
 
+    fn mark_tx_trusted(&mut self, _txid: TxId) -> Result<(), Self::Error> {
+        todo!()
```

Comment:
Is it ok to leave this as a `todo!()`?

Reply:
Yeah, this is in `zcash_client_memory` which ChainSafe has a grant to work on support for.

Reply:
Could we put a note about that in here? Either in comments or in the todo itself: `todo!("to be implemented in grant work by ChainSafe")`.